tests/Aspire.Hosting.LocalStack.Integration.Tests/Aspire.Hosting.LocalStack.Integration.Tests.csproj

```diff
@@ -1,14 +1,18 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <Sdk Name="Aspire.AppHost.Sdk" Version="9.4.0" />
```

Comment:
The changes in this file is to allow inline creation of `DistributedApplicationTestingBuilder`, without a separate AppHost project.